docs/bug_bash.md

```diff
@@ -0,0 +1,98 @@
+
+
+EKS Anywhere curated packages is a framework to manage installation, configuration and maintenance of components that provide general operational capabilities for Kubernetes applications. This guide describes the steps to install EKS Anywhere package controller and install, configure, upgrade and delete a package.
+
+## Before you start
+
+You must create a EKS Anywhere cluster The easiest thing to do is create a [local EKS Anywhere cluster](https://anywhere.eks.amazonaws.com/docs/getting-started/local-environment/) on your Mac or Linux machine using the Docker provider.
+
+## Install package controller
+
+Right now, EKS Anywhere Packages is installed using a helm chart. 
+
+         helm install cert-manager oci://public.ecr.aws/j0a1m4z9/aws/eks-anywhere-packages --version v0.1.0
```

Comment:
cert-manager as a name will be confusing. let's name it "packages" or "package-manager"

Reply:
```suggestion
         helm install package-manager oci://public.ecr.aws/j0a1m4z9/aws/eks-anywhere-packages --version v0.1.0
```

Copy pasta mistake